cockatrice/src/player.cpp

```diff
@@ -2514,16 +2514,21 @@ void Player::actResetPT()
     QListIterator<QGraphicsItem *> selected(scene()->selectedItems());
     while (selected.hasNext()) {
         auto *card = static_cast<CardItem *>(selected.next());
-        CardInfoPtr info = card->getInfo();
-        if (!info) {
+        QString ptString;
+        if (!card->getFaceDown()) { // leave the pt empty if the card is face down
+            CardInfoPtr info = card->getInfo();
+            if (info) {
+                ptString = info->getPowTough();
+            }
+        }
+        if (ptString == card->getPT()) {
```

Comment:
Empty strings are counted as 0/0 and are generally more intuitive than hiding -1/0.
In all relevant cases an empty card info means that the card is simply unknown to the player, this also means that if a player doesn't have a card it will now be reset instead.